API/RecommendationService/cosmos_helper.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+from azure.cosmos import CosmosClient
+
+from .util import generated_query_kql
+
+client = CosmosClient(os.environ["CosmosDB_Endpoint"], os.environ["CosmosDB_Key"])
+database = client.get_database_client(os.environ["CosmosDB_DataBase"])
+knowledge_base_container = database.get_container_client(os.environ["KnowledgeBase_Container"])
+recommendation_container = database.get_container_client(os.environ["Recommendation_Container"])
+recommendation_container_2 = database.get_container_client(os.environ["Recommendation_Container_2"])
+e2e_scenario_container = database.get_container_client(os.environ["E2EScenario_Container"])
```

Comment:
The current connection objects are all single instance, which may affect the concurrency. When the concurrent requests to recommendation service are high, we can consider optimizing it to the connection pool